src/resources/postcss/views/day/_event.pcss

```diff
@@ -0,0 +1,86 @@
+.tribe-events {
+
+	/* -----------------------------------------------------------------------------
+	 * Event
+	 * ----------------------------------------------------------------------------- */
+
+	.tribe-events-calendar-day__event {
+		display: flex;
+		flex-direction: column;
+		margin-bottom: var(--spacer-5);
```

Comment:
same notes as https://github.com/moderntribe/the-events-calendar/pull/2585 on these spacers, caution when merging as you'll likely have conflicts.